### PR TITLE
remove allOf attribute

### PR DESCRIPTION
**Pull request by max-weis**
allOf Attribute musste Auskommentiert werden, da es nicht korrekt geschrieben wird

### CODE REVIEW

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java [hunk 1]
```diff
@@ -221,8 +220,8 @@ public static Schema readSchema(final AnnotationScannerContext context, Schema s
                 type -> readClassSchemas(context, type), defaults));
         schema.setAnyOf(SchemaFactory.<Type[], List<Schema>> readAttr(annotation, SchemaConstant.PROP_ANY_OF,
                 type -> readClassSchemas(context, type), defaults));
-        schema.setAllOf(SchemaFactory.<Type[], List<Schema>> readAttr(annotation, SchemaConstant.PROP_ALL_OF,
-                type -> readClassSchemas(context, type), defaults));
+        //schema.setAllOf(SchemaFactory.<Type[], List<Schema>> readAttr(annotation, SchemaConstant.PROP_ALL_OF,
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:223]: please remove comments

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java [hunk 2]
```diff
@@ -177,7 +177,7 @@ public static Schema readSchema(final JsonNode node) {
         schema.setEnumeration(JsonUtil.readStringArray(node.get(SchemaConstant.PROP_ENUMERATION)).orElse(null));
         schema.setType(readSchemaType(node.get(SchemaConstant.PROP_TYPE)));
         schema.setNot(readSchema(node.get(SchemaConstant.PROP_NOT)));
-        schema.setAllOf(readSchemaArray(node.get(SchemaConstant.PROP_ALL_OF)).orElse(null));
+        //schema.setAllOf(readSchemaArray(node.get(SchemaConstant.PROP_ALL_OF)).orElse(null));
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java:180]: please remove comments

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java [hunk 3]
```diff
@@ -236,7 +236,7 @@ private static SchemaProperty readSchemaProperty(final JsonNode node) {
         schema.setEnumeration(JsonUtil.readStringArray(node.get(SchemaConstant.PROP_ENUMERATION)).orElse(null));
         schema.setType(readSchemaType(node.get(SchemaConstant.PROP_TYPE)));
         //schema.setNot(readSchema(node.get(SchemaConstant.PROP_NOT)));
-        schema.setAllOf(readSchemaArray(node.get(SchemaConstant.PROP_ALL_OF)).orElse(null));
+        //schema.setAllOf(readSchemaArray(node.get(SchemaConstant.PROP_ALL_OF)).orElse(null));
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java:239]: please remove comments

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaWriter.java [hunk 4]
```diff
@@ -149,7 +149,7 @@ public static void writeSchema(ObjectNode node, Schema model) {
             ObjectWriter.writeStringArray(node, model.getRequired(), SchemaConstant.PROP_REQUIRED);
             ObjectWriter.writeStringArray(node, model.getEnumeration(), SchemaConstant.PROP_ENUMERATION);
             JsonUtil.enumProperty(node, SchemaConstant.PROP_TYPE, model.getType());
-            writeSchemaList(node, model.getAllOf(), SchemaConstant.PROP_ALL_OF);
+            //writeSchemaList(node, model.getAllOf(), SchemaConstant.PROP_ALL_OF);
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaWriter.java:152]: please remove comments

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java [hunk 5]
```diff
@@ -255,19 +254,19 @@ public static Schema readSchema(final AnnotationScannerContext context, Schema s
...
-                Map<String, Schema> propertySchemas = new LinkedHashMap<>(properties.length);
-                for (AnnotationInstance propAnnotation : properties) {
-                    String key = JandexUtil.value(propAnnotation, SchemaConstant.PROP_NAME);
-                    Schema value = readSchema(context, new SchemaImpl(), propAnnotation, Collections.emptyMap());
-                    propertySchemas.put(key, value);
-                }
-
-                return propertySchemas;
-            }, defaults));
+                SchemaConstant.PROP_PROPERTIES, properties -> {
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:257]: please check indentation is wanted or unwanted

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java [hunk 6]
```diff
@@ -135,7 +135,7 @@ private static SchemaType readSchemaType(final JsonNode node) {
              * {@link org.eclipse.microprofile.openapi.annotations.Components}.
              */
             if (name != null) {
-              map.put(name, SchemaFactory.readSchema(context, new SchemaImpl(name), nested, Collections.emptyMap()));
+                map.put(name, SchemaFactory.readSchema(context, new SchemaImpl(name), nested, Collections.emptyMap()));
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaReader.java:138]: please check if indentation is wanted or unwanted

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java [hunk 7]
```diff
@@ -409,14 +406,14 @@ static Schema readClassSchema(final AnnotationScannerContext context, Type type,
     }
 
     private static Schema otherTypeToSchema(final AnnotationScannerContext context, Type type,
-        List<AnnotationScannerExtension> extensions) {
+            List<AnnotationScannerExtension> extensions) {
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:409]: please check if indentation is wanted or unwanted

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java [hunk 8]
```diff
@@ -409,14 +406,14 @@ static Schema readClassSchema(final AnnotationScannerContext context, Type type,
...
 
         Type asyncType = resolveAsyncType(context, type, extensions);
         return schemaRegistration(context, asyncType, AsyncApiDataObjectScanner.process(context, asyncType));
     }
 
     static Type resolveAsyncType(final AnnotationScannerContext context, Type type,
-        List<AnnotationScannerExtension> extensions) {
+            List<AnnotationScannerExtension> extensions) {
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:416]: please check if indentation is wanted or unwanted

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java [hunk 9]
```diff
@@ -313,7 +312,7 @@ public static Schema readSchema(final AnnotationScannerContext context, Schema s
      * @return Schema model
      */
     public static Schema typeToSchema(final AnnotationScannerContext context, Type type,
-        List<AnnotationScannerExtension> extensions) {
+            List<AnnotationScannerExtension> extensions) {
```
Comment (stephan-mueller) [inline core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:315]: please check if indentation is wanted or unwanted

**Review by stephan-mueller — CHANGES_REQUESTED** [review]
Please check my comments

File: core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java (hunk 5 above)
Comment (max-weis) [reply core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java:257]: it is

**Review by stephan-mueller — APPROVED** [review]

**Pull request merged**